pom.xml

```diff
@@ -299,8 +299,8 @@
           <version>1.1.0</version>
           <configuration>
             <cdapArtifacts>
-              <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-              <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+              <parent>system:cdap-data-pipeline2_2.11[6.8.0,7.0.0-SNAPSHOT)</parent>
```

Comment:
this should still be cdap-data-pipeline, not cdap-data-pipeline2_2.11.

It's a bit confusing, but the name here refers to the CDAP artifact name and not a maven artifactId.